sys-apps/pciutils/pciutils-3.6.0.recipe

```diff
@@ -14,13 +14,17 @@ HOMEPAGE="https://mj.ucw.cz/sw/pciutils/"
 COPYRIGHT="1997-2018 Martin Mares"
 LICENSE="GNU GPL v2"
 REVISION="1"
-SOURCE_URI="ftp://atrey.karlin.mff.cuni.cz/pub/linux/pci/pciutils-$portVersion.tar.gz"
+SOURCE_URI="ftp://atrey.karlin.mff.cuni.cz/pub/linux/pci/pciutils-$portVersion.tar.gz
+	https://mirrors.edge.kernel.org/pub/software/utils/pciutils/pciutils-$portVersion.tar.gz"
```

Comment:
Oops, the tarball we get from the second URI does not have the same sha256sum:
```
36453246969d1fe2ec8da9db481ed12f288f3e9117d24caf10238cb3cfb8e58f  pciutils-3.6.0.tar.gz
```
This means we can't keep this mirror.
I found https://www.mirrorservice.org/pub/software/utils/pciutils/ but it has the same sha256sum as the above mirror, so we can't use it either.

The only mirror we can use is https://fossies.org/linux/misc/pciutils-3.6.0.tar.gz/

```diff
 SOURCE_URI="ftp://atrey.karlin.mff.cuni.cz/pub/linux/pci/pciutils-$portVersion.tar.gz
-	https://mirrors.edge.kernel.org/pub/software/utils/pciutils/pciutils-$portVersion.tar.gz"
+	https://fossies.org/linux/misc/pciutils-$portVersion.tar.gz"
```

Reply:
Or we can actually migrate to kernel.org mirror, they have `.xz` tarballs ;)

Reply:
Yup, this might be an option.
That said, there is (or was?) an implicit recommendation to avoid `tar.xz` tarballs for recipes that build on x86_gcc2 (primary arch). This is (or was) supposed to make the recipe friendly to users of x86_gcc2 *non*-hybrid systems, as xz_utils is not available on these systems.

So, unless someone tells us we can ignore this recommendation, I'd keep the `tar.gz` for now.
(Once gcc2 support is gone we will be free to switch to `tar.xz` :)